content/src/content/jcr_root/apps/core/wcm/components/documentcloudviewer/v1/documentcloudviewer/documentcloudviewer.html

```diff
@@ -0,0 +1,46 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2020 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<div class="cmp-documentcloudviewer" 
+     data-sly-use.documentCloudViewerModel="com.adobe.cq.wcm.core.components.models.DocumentCloudViewer"
+     data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component">
+
+     <style>
+          .cmp-documentcloudviewer__full-window, .cmp-documentcloudviewer__sized-container {
+               height: ${documentCloudViewerModel.viewerHeight @ context='styleToken'} !important;
+          }
+     </style>
+
+     <h2 data-sly-test="${!documentCloudViewerModel.documentPath}">Please select a document to display.</h2>
+     <h2 data-sly-test="${!documentCloudViewerModel.clientId}">
+          Please setup Context Aware Configuration 
+          (com.adobe.cq.wcm.core.components.internal.services.documentcloudviewer.DocumentCloudViewerCaConfig) in /conf 
+          and link it using sling:configRef on any current/parent node.</h2>
```

Comment:
The text should be internationalised.

Reply:
Fixed in c0ee0fc02e86b4ef58d805b90254b002c049deb4